### PR TITLE
Enable mpulse on live

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves #NUMBER

**Overall change:**

Enables the mPulse beacon on live

**Code changes:**

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by oluoluoxenfree — APPROVED** [review]

**Review by jamesdonoh — APPROVED** [review]
Yes please!

File: envConfig/live.env [hunk 1]
```diff
@@ -3,6 +3,7 @@ SIMORGH_ASSETS_MANIFEST_PATH=build/assets-live.json
 SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN=https://news.files.bbci.co.uk
 SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/include/articles/public
 SIMORGH_ATI_BASE_URL=https://a1.api.bbc.co.uk/hit.xiti?
+SIMORGH_MPULSE_API_KEY=JLSEX-R76DK-JXL3A-UB7YH-NPSEW
```
Comment (DemonusPC) [inline envConfig/live.env:6]: Emm, shouldn't this be a secret?

Comment (chris-hinds) [reply envConfig/live.env:6]: Nope. The API key is public. Akamai limit by domain on their end.

Comment (jamesdonoh) [reply envConfig/live.env:6]: It's accessible on the client by necessity so cannot really be a secret.

Comment (DemonusPC) [reply envConfig/live.env:6]: Alright dab it is

**Review by DemonusPC — APPROVED** [review]
Dank

**Comment by chris-hinds** [comment]
This doesn't need testing.

**Pull request merged**